conda/meta.yaml

```diff
@@ -27,6 +27,7 @@ requirements:
     - python
     - matplotlib
     - ipywidgets
+    - ginsim-python
```

Comment:
We can avoid having a strong requirement on ginsim, see my comments below.

Reply:
Done